packages/dds/tree/src/test/util/summarySizeTests.spec.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { getSummaryBenchmarkSizes } from "./summaryBenchmarkSizes";
```

Comment:
I don't understand the motivation for splitting this up into two files. Do we expect anything else to use the exports from that file?

Reply:
Combined the files into 1